src/utils/expandedLines/expandedLines.test.ts

```diff
@@ -11,6 +11,16 @@ describe("mergeIntervals", () => {
     ).toStrictEqual([[89, 102]]);
   });
 
+  it("merges overlapping intervals when end + 1 > start", () => {
+    expect(
+      mergeIntervals([
+        [0, 5],
+        [3, 15],
+        [7, 28],
+      ])
+    ).toStrictEqual([[0, 28]]);
```

Comment:
What scenario would this occur in? I'm glad we are handling this but I'm wondering if there is a case where this could happen.

Reply:
in theory, it should never happen. But I figured it would be good to cover all bases, especially since it has no impact on current functionality :]